index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Page</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header>
+            <h1 class="header">John Doe</h1>
+    
+            <nav>
+                <img src="assets/images/menuIcon.png" alt="" class="icon menu-icon">
+                <div class="nav-links" style="display: none;">
+                    <a href="" id="portfolio">Portfolio</a>
+                    <a href="" id="about">About</a>
+                    <a href="" id="contact">Contact</a>
+                </div>
+            </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <span class="header">Hey There.
+                I’m Alex</span>
+            <span class="sub-header">I’am a Software Developer</span>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <div class="social-icons">
+                <a href="https://github.com/Reggeditt" class="social-link">
+                    <img src="assets/images/githubIcon.png" alt="" class="icon">
+                </a>
+                <a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
+                    <img src="assets/images/linkedin.png" alt="" class="icon">
+                </a>
+                
+                <a href="https://twitter.com/tksarso" class="social-link">
+                    <img src="assets/images/twitterIcon.png" alt="" class="icon">
+                </a>
+            </div>
+            <figure>
+                <img src="assets/images/Illustration.png" alt="" class="artwork">
+            </figure>
```

Comment:
- [ ] Good job adding the image as the previous reviewer has earlier suggested, I think it will be better if the image is a background image added from CSS and not in the `<figure>` tag because the image is a decoration, and decorations are always background-images, not `img` tags.  

When an image does not add to the meaning of the content in a webpage, you add such images as a background image 👍🏽

Reply:
Alright thanks!

Reply:
done!